core-build/gulp-core-build/src/index.ts

```diff
@@ -45,16 +45,16 @@ export * from './tasks/CleanTask';
 export * from './tasks/ValidateShrinkwrapTask';
 export * from './jsonUtilities/SchemaValidator';
 
-// tslint:disable-next-line:no-any
-const packageJSON: any = require(path.resolve(process.cwd(), 'package.json'));
 const _taskMap: { [key: string]: IExecutable } = {};
 const _uniqueTasks: IExecutable[] = [];
 
-const packageFolder: string =
-  (packageJSON.directories && packageJSON.directories.packagePath) ?
-    packageJSON.directories.packagePath : '';
+const packageFolder: string = (builtPackage.directories && builtPackage.directories.packagePath)
+  ? builtPackage.directories.packagePath
+  : '';
 
 let _buildConfig: IBuildConfig = {
+  gulp: undefined as any, // tslint:disable-line:no-any
```

Comment:
concur.

---
In reply to: [124909572](https://github.com/Microsoft/web-build-tools/pull/256#discussion_r124909572) [](ancestors = 124909572)